notebooks/book2/02/sub_super_gauss_plot.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>For all these plots, use set aspect to equal: https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.axes.Axes.set_aspect.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
If I apply an equal aspect ratio to the first figure, I am getting this output
![1D_latexified](https://user-images.githubusercontent.com/52617262/174956694-1b710919-e9d0-4d49-b96d-ef03350b9aca.png)
.

Reply:
<p>Equal aspect is needed for the final three figures and not for this one :)</p>